webapp-django/crashstats/signature/views.py

```diff
@@ -378,8 +378,6 @@ def signature_correlations(request, params):
     elif 'version' in params and params['version']:
         if all('b' in version for version in params['version']):
             context['channel'] = 'beta'
-        elif all('a2' in version for version in params['version']):
-            context['channel'] = 'aurora'
         elif all('a1' in version for version in params['version']):
```

Comment:
The only way this could still be needed is if someone pulls up the Correlations tab (under Signature Report), and adds additional filtering on specifically an old "XX.0a2" version. E.g:
<img width="1540" alt="screen shot 2017-07-28 at 10 24 02 am" src="https://user-images.githubusercontent.com/26739/28721593-e808c226-737e-11e7-8b73-fcf04260cbf1.png">
But mind you, any of those XX.0a2 versions have to be fully "manually" typed in since it's not an option from the dropdown suggestions that appear in that input field.